delete_file_h3113.list

```diff
@@ -0,0 +1,2 @@
+/etc/ofono/ril_subscription.d/dual-sim.conf
+/usr/share/ssu/board-mappings.d/10-h4113.ini
```

Comment:
A comment for the future: Looks like this could benefit from a script, whenever more devices will get added (e.g. all variants of discovery)
Cc @Thaodan

Reply:
Can we move that in a seperate package? I don't like the idea of delete lists that much except for workarounds. Like droid-configs-nile-h4x13?

Explanation:
All dual sim device are 4x13
All single devices are 3x13
6" devices screen are all: x213
5.2" devices are x113.

That avoids a script at all and would work for the 6" version too.

Reply:
Droid config includes config files and I am afraid that we need to have own package for each device/variants of it. I am not sure what you mean by separate package? You mean that we should have own package for each ssu config and dual sim config for ofono?